### PR TITLE
User-set titles for liquid-core, rocketchat, nextcloud and dokuwiki

### PR DESCRIPTION
**Pull request by gabriel-v**
rocket
doku

Nothing for Hoover at the moment...
hoover

see https://github.com/liquidinvestigations/liquid-nextcloud/pull/2
and https://github.com/liquidinvestigations/core/pull/49

### CODE REVIEW

File: examples/liquid.ini [hunk 1]
```diff
@@ -11,6 +11,11 @@ consul_socket = ../cluster/var/consul.socket
 # Domain for the liquid bundle.
 domain = liquid.example.org
 
+# User-friendly title.
+# Defaults to the domain with no spaces, capitalized.
```
Comment (mgax) [inline examples/liquid.ini]: How about we add more magic and remove the last bit of the domain name? That would actually yield more relevant vaues.

File: templates/nextcloud-migrate.nomad [hunk 2]
```diff
@@ -15,7 +15,7 @@ job "nextcloud-migrate" {
 
       driver = "docker"
       config = {
-        image = "liquidinvestigations/liquid-nextcloud"
+        image = "liquidinvestigations/liquid-nextcloud:title"
```
Comment (mgax) [inline templates/nextcloud-migrate.nomad]: Don't forget to remove these.

File: templates/rocketchat.nomad [hunk 3]
```diff
@@ -69,16 +69,16 @@ job "rocketchat" {
             ADMIN_PASS={{.Data.pass}}
           {{- end }}
           ADMIN_EMAIL=admin@example.com
-          Organization_Name=Liquid
-          Site_Name=Liquid
+          Organization_Name=${config.liquid_title} Liquid
```
Comment (mgax) [inline templates/rocketchat.nomad]: Why keep a trailing "Liquid" here?

Comment (gabriel-v) [reply templates/rocketchat.nomad]: removed

File: templates/nextcloud-migrate.nomad (hunk 2 above)
Comment (gabriel-v) [reply templates/nextcloud-migrate.nomad]: done

File: examples/liquid.ini (hunk 1 above)
Comment (mgax) [reply examples/liquid.ini]: overkill

**Pull request merged**